backend/dal/Migrations/20210125213804_v01.07.00.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+using Pims.Dal.Helpers.Migrations;
+
+namespace Pims.Dal.Migrations
+{
+    public partial class v010700 : SeedMigration
```

Comment:
We are planning to release `v01.06.00` this sprint, as we were unable to release it last sprint.  So you'll probably need to update to be `v01.06.02` or `v01.06.03` depending on whether @devinleighsmith merges his PR before yours.